egs2/aphasiabank/asr1/local/clean_hyp_annotations.py

```diff
@@ -0,0 +1,38 @@
+"""
+Remove [APH] and [NONAPH] tags from the hypothesis file.
```

Comment:
Given that you already registered APH and NONAPH into nlsyms.txt, it is necessary to have this script.

Reply:
My bad, I forgot to mention that `--nlsyms_txt` needs to be unset during scoring to keep the tags.
By doing this, we can calculate the WER of overall, patients', and control group's subset.

Reply:
Added instructions in README.md